packages/components-react/heading-6-react/src/heading-6.tsx

```diff
@@ -0,0 +1,15 @@
+import { clsx } from 'clsx';
+import type { HTMLAttributes } from 'react';
+import { forwardRef } from 'react';
+
+type Heading6Props = HTMLAttributes<HTMLHeadingElement>;
```

Comment:
De type of `interface` moet geexporteerd worden, da's handig

Reply:
Fixed